src/main/kotlin/model/NDH.kt

```diff
@@ -63,7 +63,7 @@ class NDH private constructor() {
     }
 
     fun defineDirectRelationships() {
-      system.uses(NOMIS.custodyApi, "periodically polls the NOMIS Events table to look for changes to offender data via")
+      system.uses(NOMIS.system, "periodically polls the NOMIS Events table to look for changes to offender data via", "Oracle XTAG queue")
```

Comment:
The Oracle XTAG queue is consumed by a weblogic instance which then send events to the NDH.  @Mjwillis - can you provide any more information on how this works?